### PR TITLE
Configure coredumps + adapt debugging mechanisms

### PR DESCRIPTION
**Pull request by k0ch4lo**

### CODE REVIEW

File: recipes-initscripts/cml-boot/cml-boot_1.0.bb [hunk 1]
```diff
@@ -47,8 +47,16 @@ do_install:append () {
 	if [ "y" = "${DEVELOPMENT_BUILD}" ];then
 		bbwarn "Patching /init script to start SSH server in cml layer"
 		sed -i '\|#DEV_START_SSHD#|e cat ${WORKDIR}/start_sshd' ${D}/init
+
+		bbwarn "Patching /init script to mount external data fs for debugging purposes"
```
Comment (jwsbck) [inline recipes-initscripts/cml-boot/cml-boot_1.0.bb:51]: perhaps add half a sentence in the description above, so this is not only sshd anymore?

File: images/trustx-cml-initramfs.bb [hunk 2]
```diff
@@ -69,13 +74,14 @@ update_tabs () {
 
     mkdir -p ${IMAGE_ROOTFS}/dev
     mknod -m 622 ${IMAGE_ROOTFS}/dev/tty12 c 4 12
+
+	bbwarn "Enabling external partition for /data/cml/containers for debugging purposes"
+	echo 'LABEL=containers     /data/cml/containers ext4       nosuid,nodev,noexec,nofail 0  0' >> ${IMAGE_ROOTFS}/etc/fstab
```
Comment (jwsbck) [inline images/trustx-cml-initramfs.bb:79]: consistent indentation

**Review by jwsbck — CHANGES_REQUESTED** [review]
only nitpicking

File: images/trustx-cml-initramfs.bb (hunk 2 above)
Comment (k0ch4lo) [reply images/trustx-cml-initramfs.bb:82]: change whitespace to tabs consistently

File: recipes-initscripts/cml-boot/cml-boot_1.0.bb (hunk 1 above)
Comment (k0ch4lo) [reply recipes-initscripts/cml-boot/cml-boot_1.0.bb:52]: adapted the comment reflecting the changes

**Review by jwsbck — APPROVED** [review]

**Pull request merged**